doc/index.md

```diff
@@ -1,100 +1,178 @@
 # Introduction
 
-lua-http is an HTTP library for Lua, it supports: both client and server operations, both HTTP 1 and HTTP 2.
+Lua-http is an performant, capable Hyper Text Transfer Protocol (HTTP) and WebSocket (WS) library for Lua 5.1, 5.2, 5.3 and LuaJIT. Lua-http can be run in either a Server (listen) or Client (connect) mode and includes the following features: 
+- HTTP 1 and HTTP 2 as specified by RFCs [1945](https://tools.ietf.org/html/rfc1945) and [7540 ](https://tools.ietf.org/html/rfc7540)
```

Comment:
RFC 1945 was not used as reference material at all. HTTP 1 is specified in mostly in RFC 7230 (also RFC 7231, but really all RFCs from 7230 to 7240); I did have to refer to the historical standard of RFC 2616 for a couple of minor things, but I wouldn't add that to the docs.